server/server.go

```diff
@@ -171,3 +176,55 @@ func (server *HTTPServer) Start() error {
 func (server *HTTPServer) Stop(ctx context.Context) error {
 	return server.Serv.Shutdown(ctx)
 }
+
+// genericHandle
+func (server HTTPServer) genericAggregatorRedirect(writer http.ResponseWriter, request *http.Request) {
+	endpointComps := strings.Split(request.RequestURI, server.Config.APIPrefix)
```

Comment:
maybe you meant strings.TrimPrefix?

Reply:
The result in case where the prefix is missing will be slightly different, but I agree that `TrimPrefix` is a safer choice here.

Reply:
I'm new in strings handling in Go, so probably yes. TYVM